### PR TITLE
WebPA recipe support + Bump thunder/plugins to latest

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: recipes-support/libtinyxml/libtinyxml_2.6.2.bbappend [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```
Comment (wouterlucas) [inline recipes-support/libtinyxml/libtinyxml_2.6.2.bbappend:1]: who provides libtinyxml?

Comment (HaseenaSainul) [reply recipes-support/libtinyxml/libtinyxml_2.6.2.bbappend:1]: Its available in meta-openembedded/meta-oe/recipes-support

Comment (wouterlucas) [reply recipes-support/libtinyxml/libtinyxml_2.6.2.bbappend:1]: In that case should we upstream the package config changes?

Comment (HaseenaSainul) [reply recipes-support/libtinyxml/libtinyxml_2.6.2.bbappend:1]: The version currently included by meta-oe does not have this package config. But the higher version of the libtinyxml package have the package config support already (even with 2.6.2_2 itself have the support)

Comment (wouterlucas) [reply recipes-support/libtinyxml/libtinyxml_2.6.2.bbappend:1]: I understand that, what I'm asking is should we send a patch to meta-openembedded morty to update the community with a similar change versus keeping it in meta-wpe?

Comment (HaseenaSainul) [reply recipes-support/libtinyxml/libtinyxml_2.6.2.bbappend:1]: But currently RDK is using the its own fork of meta-openembedded from :https://code.rdkcentral.com/r/rdk/components/opensource/oe/meta-openembedded with rdk/morty branch. So in that case, pushing the changes to meta-openembedded:morty will not work. So do we need to push this to rdk/morty branch of rdk openembedded fork ?

**Pull request merged**